### DISCUSSION TITLE
Where can I read about _authOrigins?

### DISCUSSION THREAD

**Discussion by twiddlecoding** [pocketbase/pocketbase#6076] [Q&A]
Hi!
Many thanks for everyone who contributed to the latest major update.
I have tried to find information regarding the collection _authOrigins. What does it do? Where can I read about it?

**Comment by ganigeorgiev** [comment]
It is a system collection and most users shouldn't be concerned with it.

When an auth collection has the "Send email alert for new login" option enabled, in the `_authOrigins` collection we store as md5 hash (currently IP + user agent) information about the user login to determine whether to send an alert or not.

**Reply by twiddlecoding** [reply]
Oh, I see. I thought it might have been a new way of validating that a certain user is logged in.

**Reply by ganigeorgiev** [reply]
We don't store the tokens in the database out of the box and there are no "sessions" in the traditional sense, aka. the API is fully stateless and it relies each request to have valid `Authorization:TOKEN` header with it in order a single user to be considered "authenticated".
